src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,69 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileGotRead = readFromFile(fromFileName);
+        String[] dividedByEmptySpace = fileGotRead.split(" ");
+        int supplySum = getSupplySum(dividedByEmptySpace);
+        int buySum = getBuySum(dividedByEmptySpace);
+        int benefit = supplySum - buySum;
```

Comment:
encapsulate this logic in `generateReport` method which would return a string of a report